src/interfaces/vault.spec.ts

```diff
@@ -710,28 +720,50 @@ describe("VaultInterface", () => {
         });
 
         describe("when token is not eth address", () => {
-          it("should deposit into a yearn vault", async () => {
-            const assetStaticVaultV2 = createMockAssetStaticVaultV2({ token: "0xToken" });
-            vaultInterface.getStatic = jest.fn().mockResolvedValue([assetStaticVaultV2]);
+          describe("when there is no partner service", () => {
+            it("should deposit directly into a yearn vault", async () => {
+              const assetStaticVaultV2 = createMockAssetStaticVaultV2({ token: "0xToken" });
+              vaultInterface.getStatic = jest.fn().mockResolvedValue([assetStaticVaultV2]);
 
-            const executeVaultContractTransactionMock = jest.fn().mockResolvedValue("trx");
-            (vaultInterface as any).executeVaultContractTransaction = executeVaultContractTransactionMock;
+              const executeVaultContractTransactionMock = jest.fn().mockResolvedValue("trx");
+              (vaultInterface as any).executeVaultContractTransaction = executeVaultContractTransactionMock;
 
-            const [vault, token, amount, account] = ["0xVault", "0xToken", "1", "0xAccount"];
+              const [vault, token, amount, account] = ["0xVault", "0xToken", "1", "0xAccount"];
 
-            const actualDeposit = await vaultInterface.deposit(vault, token, amount, account);
+              const actualDeposit = await vaultInterface.deposit(vault, token, amount, account);
 
-            expect(Contract).toHaveBeenCalledTimes(1);
-            expect(Contract).toHaveBeenCalledWith(
-              "0xVault",
-              ["function deposit(uint256 amount) public", "function withdraw(uint256 amount) public"],
-              {
-                sendTransaction: sendTransactionMock
-              }
-            );
-            expect(executeVaultContractTransactionMock).toHaveBeenCalledTimes(1);
-            expect(executeVaultContractTransactionMock).toHaveBeenCalledWith(expect.any(Function), {});
-            expect(actualDeposit).toEqual("trx");
+              expect(Contract).toHaveBeenCalledTimes(1);
+              expect(Contract).toHaveBeenCalledWith(
+                "0xVault",
+                ["function deposit(uint256 amount) public", "function withdraw(uint256 amount) public"],
+                {
+                  sendTransaction: sendTransactionMock
+                }
+              );
+              expect(executeVaultContractTransactionMock).toHaveBeenCalledTimes(1);
+              expect(executeVaultContractTransactionMock).toHaveBeenCalledWith(expect.any(Function), {});
```

Comment:
I would add here an expectation to be sure the partner logic is not being called, e.g.
```typescript
expect(partnerPopulateDepositTransactionMock).not.toHaveBeenCalled();
```

Reply:
Good suggestion, will do